src/main/java/fr/foodlimit/api/security/MockSecurityConfig.java

```diff
@@ -0,0 +1,48 @@
+package fr.foodlimit.api.security;
+
+import fr.foodlimit.api.security.jwt.JWTConfigurer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'fr.foodlimit.api.security.jwt.JWTConfigurer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)